<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="4" failed="4" total="10" passed="2">
  <reporter-output>
    <line>
      <![CDATA[Homepage validated ]]>
    </line>
    <line>
      <![CDATA[Login navigates to Login page  ]]>
    </line>
    <line>
      <![CDATA[Sign up navigates to Signup page  ]]>
    </line>
    <line>
      <![CDATA[Test Passed: Home Page Validation]]>
    </line>
    <line>
      <![CDATA[Administrator Login is Successful]]>
    </line>
    <line>
      <![CDATA[Administrator Logout is Successful]]>
    </line>
    <line>
      <![CDATA[Test Passed: Administrator Login-Logout ]]>
    </line>
    <line>
      <![CDATA[Student Login is Succesful]]>
    </line>
  </reporter-output>
  <suite name="Suite" duration-ms="7791538" started-at="2015-12-11T07:43:15Z" finished-at="2015-12-11T09:53:07Z">
    <groups>
    </groups>
    <test name="Test" duration-ms="7791538" started-at="2015-12-11T07:43:15Z" finished-at="2015-12-11T09:53:07Z">
      <class name="com.vedantu.test.sanity1_testsuite.AdminLogin">
        <test-method status="PASS" signature="preCondition()[pri:0, instance:com.vedantu.test.sanity1_testsuite.AdminLogin@1a407d53]" name="preCondition" is-config="true" duration-ms="15378" started-at="2015-12-11T13:13:44Z" finished-at="2015-12-11T13:14:00Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- preCondition -->
        <test-method status="PASS" signature="AdministartorLogin()[pri:0, instance:com.vedantu.test.sanity1_testsuite.AdminLogin@1a407d53]" name="AdministartorLogin" duration-ms="13572" started-at="2015-12-11T13:14:00Z" finished-at="2015-12-11T13:14:13Z">
          <reporter-output>
            <line>
              <![CDATA[Administrator Login is Successful]]>
            </line>
            <line>
              <![CDATA[Administrator Logout is Successful]]>
            </line>
            <line>
              <![CDATA[Test Passed: Administrator Login-Logout ]]>
            </line>
          </reporter-output>
        </test-method> <!-- AdministartorLogin -->
        <test-method status="PASS" signature="postCondition()[pri:0, instance:com.vedantu.test.sanity1_testsuite.AdminLogin@1a407d53]" name="postCondition" is-config="true" duration-ms="956" started-at="2015-12-11T13:14:13Z" finished-at="2015-12-11T13:14:14Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- postCondition -->
      </class> <!-- com.vedantu.test.sanity1_testsuite.AdminLogin -->
      <class name="com.vedantu.test.sanity1_testsuite.HomePageValidation">
        <test-method status="PASS" signature="preCondition()[pri:0, instance:com.vedantu.test.sanity1_testsuite.HomePageValidation@21bcffb5]" name="preCondition" is-config="true" duration-ms="26341" started-at="2015-12-11T13:13:15Z" finished-at="2015-12-11T13:13:42Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- preCondition -->
        <test-method status="PASS" signature="HomePageValidations()[pri:0, instance:com.vedantu.test.sanity1_testsuite.HomePageValidation@21bcffb5]" name="HomePageValidations" duration-ms="2443" started-at="2015-12-11T13:13:42Z" finished-at="2015-12-11T13:13:44Z">
          <reporter-output>
            <line>
              <![CDATA[Homepage validated ]]>
            </line>
            <line>
              <![CDATA[Login navigates to Login page  ]]>
            </line>
            <line>
              <![CDATA[Sign up navigates to Signup page  ]]>
            </line>
            <line>
              <![CDATA[Test Passed: Home Page Validation]]>
            </line>
          </reporter-output>
        </test-method> <!-- HomePageValidations -->
        <test-method status="PASS" signature="postCondition()[pri:0, instance:com.vedantu.test.sanity1_testsuite.HomePageValidation@21bcffb5]" name="postCondition" is-config="true" duration-ms="171" started-at="2015-12-11T13:13:44Z" finished-at="2015-12-11T13:13:44Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- postCondition -->
      </class> <!-- com.vedantu.test.sanity1_testsuite.HomePageValidation -->
      <class name="com.vedantu.test.sanity1_testsuite.TeacherLogin">
        <test-method status="FAIL" signature="preCondition()[pri:0, instance:com.vedantu.test.sanity1_testsuite.TeacherLogin@380fb434]" name="preCondition" is-config="true" duration-ms="0" started-at="2015-12-11T13:14:14Z" finished-at="2015-12-11T13:14:14Z">
          <exception class="org.openqa.selenium.remote.UnreachableBrowserException">
            <message>
              <![CDATA[Error communicating with the remote browser. It may have died.
Build info: version: '2.48.2', revision: '41bccdd10cf2c0560f637404c2d96164b67d9d67', time: '2015-10-09 13:08:06'
System info: host: '45.112.138-broadband.acttv.in', ip: '192.168.0.252', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.11.1', java.version: '1.8.0_65'
Driver info: driver.version: FirefoxDriver]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.remote.UnreachableBrowserException: Error communicating with the remote browser. It may have died.
Build info: version: '2.48.2', revision: '41bccdd10cf2c0560f637404c2d96164b67d9d67', time: '2015-10-09 13:08:06'
System info: host: '45.112.138-broadband.acttv.in', ip: '192.168.0.252', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.11.1', java.version: '1.8.0_65'
Driver info: driver.version: FirefoxDriver
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:641)
	at org.openqa.selenium.firefox.FirefoxDriver.access$100(FirefoxDriver.java:65)
	at org.openqa.selenium.firefox.FirefoxDriver$1$1.implicitlyWait(FirefoxDriver.java:244)
	at com.vedantu.test.pom.Login.preCondition(Login.java:26)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:564)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:213)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:653)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:295)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:90)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:203)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:155)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
Caused by: org.apache.http.conn.HttpHostConnectException: Connect to 127.0.0.1:7060 [/127.0.0.1] failed: Connection refused
	at org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:151)
	at org.apache.http.impl.conn.PoolingHttpClientConnectionManager.connect(PoolingHttpClientConnectionManager.java:353)
	at org.apache.http.impl.execchain.MainClientExec.establishRoute(MainClientExec.java:380)
	at org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:236)
	at org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:184)
	at org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:88)
	at org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:110)
	at org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:184)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:71)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:55)
	at org.openqa.selenium.remote.internal.ApacheHttpClient.fallBackExecute(ApacheHttpClient.java:161)
	at org.openqa.selenium.remote.internal.ApacheHttpClient.execute(ApacheHttpClient.java:89)
	at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:142)
	at org.openqa.selenium.firefox.internal.NewProfileExtensionConnection.execute(NewProfileExtensionConnection.java:170)
	at org.openqa.selenium.firefox.FirefoxDriver$LazyCommandExecutor.execute(FirefoxDriver.java:380)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:620)
	... 32 more
Caused by: java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at org.apache.http.conn.socket.PlainConnectionSocketFactory.connectSocket(PlainConnectionSocketFactory.java:74)
	at org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:134)
	... 47 more
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.remote.UnreachableBrowserException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- preCondition -->
        <test-method status="SKIP" signature="TeacherLoginTest()[pri:0, instance:com.vedantu.test.sanity1_testsuite.TeacherLogin@380fb434]" name="TeacherLoginTest" duration-ms="6" started-at="2015-12-11T15:21:09Z" finished-at="2015-12-11T15:21:09Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TeacherLoginTest -->
        <test-method status="SKIP" signature="postCondition()[pri:0, instance:com.vedantu.test.sanity1_testsuite.TeacherLogin@380fb434]" name="postCondition" is-config="true" duration-ms="0" started-at="2015-12-11T15:21:09Z" finished-at="2015-12-11T15:21:09Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- postCondition -->
      </class> <!-- com.vedantu.test.sanity1_testsuite.TeacherLogin -->
      <class name="com.vedantu.test.sanity1_testsuite.Signup">
        <test-method status="SKIP" signature="preCondition()[pri:0, instance:com.vedantu.test.sanity1_testsuite.Signup@668bc3d5]" name="preCondition" is-config="true" duration-ms="0" started-at="2015-12-11T15:21:09Z" finished-at="2015-12-11T15:21:09Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- preCondition -->
        <test-method status="SKIP" signature="GeneralSignup()[pri:0, instance:com.vedantu.test.sanity1_testsuite.Signup@668bc3d5]" name="GeneralSignup" duration-ms="1" started-at="2015-12-11T15:21:09Z" finished-at="2015-12-11T15:21:09Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- GeneralSignup -->
        <test-method status="SKIP" signature="postCondition()[pri:0, instance:com.vedantu.test.sanity1_testsuite.Signup@668bc3d5]" name="postCondition" is-config="true" duration-ms="0" started-at="2015-12-11T15:21:09Z" finished-at="2015-12-11T15:21:09Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- postCondition -->
      </class> <!-- com.vedantu.test.sanity1_testsuite.Signup -->
      <class name="com.vedantu.test.sanity1_testsuite.StudentLogin">
        <test-method status="SKIP" signature="preCondition()[pri:0, instance:com.vedantu.test.sanity1_testsuite.StudentLogin@3d8c7aca]" name="preCondition" is-config="true" duration-ms="0" started-at="2015-12-11T15:21:09Z" finished-at="2015-12-11T15:21:09Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- preCondition -->
        <test-method status="SKIP" signature="postCondition()[pri:0, instance:com.vedantu.test.sanity1_testsuite.StudentLogin@3d8c7aca]" name="postCondition" is-config="true" duration-ms="0" started-at="2015-12-11T15:21:09Z" finished-at="2015-12-11T15:21:09Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- postCondition -->
        <test-method status="SKIP" signature="StudentLoginTest()[pri:0, instance:com.vedantu.test.sanity1_testsuite.StudentLogin@3d8c7aca]" name="StudentLoginTest" duration-ms="0" started-at="2015-12-11T15:21:09Z" finished-at="2015-12-11T15:21:09Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- StudentLoginTest -->
      </class> <!-- com.vedantu.test.sanity1_testsuite.StudentLogin -->
      <class name="com.vedantu.test.sanity1_testsuite.TakeSession">
        <test-method status="FAIL" signature="TakingSession()[pri:0, instance:com.vedantu.test.sanity1_testsuite.TakeSession@5ebec15]" name="TakingSession" duration-ms="24557" started-at="2015-12-11T15:21:09Z" finished-at="2015-12-11T15:21:33Z">
          <exception class="org.openqa.selenium.remote.UnreachableBrowserException">
            <message>
              <![CDATA[Error communicating with the remote browser. It may have died.
Build info: version: '2.48.2', revision: '41bccdd10cf2c0560f637404c2d96164b67d9d67', time: '2015-10-09 13:08:06'
System info: host: '45.112.138-broadband.acttv.in', ip: '192.168.0.252', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.11.1', java.version: '1.8.0_65'
Driver info: driver.version: RemoteWebDriver]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.remote.UnreachableBrowserException: Error communicating with the remote browser. It may have died.
Build info: version: '2.48.2', revision: '41bccdd10cf2c0560f637404c2d96164b67d9d67', time: '2015-10-09 13:08:06'
System info: host: '45.112.138-broadband.acttv.in', ip: '192.168.0.252', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.11.1', java.version: '1.8.0_65'
Driver info: driver.version: RemoteWebDriver
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:641)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:353)
	at org.openqa.selenium.remote.RemoteWebDriver.findElementByXPath(RemoteWebDriver.java:490)
	at org.openqa.selenium.By$ByXPath.findElement(By.java:361)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:345)
	at com.vedantu.test.sanity1_testsuite.TakeSession.TakingSession(TakeSession.java:71)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:295)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:90)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:203)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:155)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
Caused by: org.apache.http.conn.HttpHostConnectException: Connect to 127.0.0.1:7055 [/127.0.0.1] failed: Connection refused
	at org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:151)
	at org.apache.http.impl.conn.PoolingHttpClientConnectionManager.connect(PoolingHttpClientConnectionManager.java:353)
	at org.apache.http.impl.execchain.MainClientExec.establishRoute(MainClientExec.java:380)
	at org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:236)
	at org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:184)
	at org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:88)
	at org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:110)
	at org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:184)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:71)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:55)
	at org.openqa.selenium.remote.internal.ApacheHttpClient.fallBackExecute(ApacheHttpClient.java:143)
	at org.openqa.selenium.remote.internal.ApacheHttpClient.execute(ApacheHttpClient.java:89)
	at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:142)
	at org.openqa.selenium.firefox.internal.NewProfileExtensionConnection.execute(NewProfileExtensionConnection.java:170)
	at org.openqa.selenium.firefox.FirefoxDriver$LazyCommandExecutor.execute(FirefoxDriver.java:380)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:620)
	... 32 more
Caused by: java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at org.apache.http.conn.socket.PlainConnectionSocketFactory.connectSocket(PlainConnectionSocketFactory.java:74)
	at org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:134)
	... 47 more
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.remote.UnreachableBrowserException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TakingSession -->
        <test-method status="FAIL" signature="TakingSession()[pri:0, instance:com.vedantu.test.sanity1_testsuite.TakeSession@5ebec15]" name="TakingSession" duration-ms="35739" started-at="2015-12-11T15:21:33Z" finished-at="2015-12-11T15:22:09Z">
          <exception class="org.openqa.selenium.WebDriverException">
            <message>
              <![CDATA[unknown error: Element is not clickable at point (524, 613). Other element would receive the click: <div class="vpopupWrapper nonner" id="v-popup" style="opacity: 0; display: block;">...</div>
  (Session info: chrome=47.0.2526.80)
  (Driver info: chromedriver=2.9.248307,platform=Mac OS X 10.11.1 x86_64) (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 62 milliseconds
Build info: version: '2.48.2', revision: '41bccdd10cf2c0560f637404c2d96164b67d9d67', time: '2015-10-09 13:08:06'
System info: host: '45.112.138-broadband.acttv.in', ip: '192.168.0.252', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.11.1', java.version: '1.8.0_65'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities [{applicationCacheEnabled=false, rotatable=false, chrome={userDataDir=/var/folders/zd/xg2w7l2d0gjdxh9ck8q3nlzr0000gn/T/.org.chromium.Chromium.Y1FSAO}, takesHeapSnapshot=true, databaseEnabled=false, handlesAlerts=true, version=47.0.2526.80, platform=MAC, browserConnectionEnabled=false, nativeEvents=true, acceptSslCerts=true, locationContextEnabled=true, webStorageEnabled=true, browserName=chrome, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true}]
Session ID: a60795750c2217bd4aa9e52df15aa0db]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.WebDriverException: unknown error: Element is not clickable at point (524, 613). Other element would receive the click: <div class="vpopupWrapper nonner" id="v-popup" style="opacity: 0; display: block;">...</div>
  (Session info: chrome=47.0.2526.80)
  (Driver info: chromedriver=2.9.248307,platform=Mac OS X 10.11.1 x86_64) (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 62 milliseconds
Build info: version: '2.48.2', revision: '41bccdd10cf2c0560f637404c2d96164b67d9d67', time: '2015-10-09 13:08:06'
System info: host: '45.112.138-broadband.acttv.in', ip: '192.168.0.252', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.11.1', java.version: '1.8.0_65'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities [{applicationCacheEnabled=false, rotatable=false, chrome={userDataDir=/var/folders/zd/xg2w7l2d0gjdxh9ck8q3nlzr0000gn/T/.org.chromium.Chromium.Y1FSAO}, takesHeapSnapshot=true, databaseEnabled=false, handlesAlerts=true, version=47.0.2526.80, platform=MAC, browserConnectionEnabled=false, nativeEvents=true, acceptSslCerts=true, locationContextEnabled=true, webStorageEnabled=true, browserName=chrome, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true}]
Session ID: a60795750c2217bd4aa9e52df15aa0db
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:422)
	at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:206)
	at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:158)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:647)
	at org.openqa.selenium.remote.RemoteWebElement.execute(RemoteWebElement.java:326)
	at org.openqa.selenium.remote.RemoteWebElement.click(RemoteWebElement.java:84)
	at com.vedantu.test.sanity1_testsuite.TakeSession.TakingSession(TakeSession.java:87)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.retryFailed(Invoker.java:1049)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1254)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:295)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:90)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:203)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:155)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.WebDriverException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TakingSession -->
        <test-method status="FAIL" signature="TakingSession()[pri:0, instance:com.vedantu.test.sanity1_testsuite.TakeSession@5ebec15]" name="TakingSession" duration-ms="31200" started-at="2015-12-11T15:22:09Z" finished-at="2015-12-11T15:22:40Z">
          <exception class="com.google.common.base.VerifyException">
            <full-stacktrace>
              <![CDATA[com.google.common.base.VerifyException
	at com.google.common.base.Verify.verify(Verify.java:98)
	at com.vedantu.test.sanity1_testsuite.TakeSession.TakingSession(TakeSession.java:102)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.retryFailed(Invoker.java:1049)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1254)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:295)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:90)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:203)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:155)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
            </full-stacktrace>
          </exception> <!-- com.google.common.base.VerifyException -->
          <reporter-output>
            <line>
              <![CDATA[Student Login is Succesful]]>
            </line>
          </reporter-output>
        </test-method> <!-- TakingSession -->
        <test-method status="FAIL" signature="TakingSession()[pri:0, instance:com.vedantu.test.sanity1_testsuite.TakeSession@5ebec15]" name="TakingSession" duration-ms="26414" started-at="2015-12-11T15:22:40Z" finished-at="2015-12-11T15:23:07Z">
          <exception class="org.openqa.selenium.WebDriverException">
            <message>
              <![CDATA[unknown error: Element is not clickable at point (524, 613). Other element would receive the click: <div class="vpopupWrapper nonner" id="v-popup" style="opacity: 0; display: block;">...</div>
  (Session info: chrome=47.0.2526.80)
  (Driver info: chromedriver=2.9.248307,platform=Mac OS X 10.11.1 x86_64) (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 44 milliseconds
Build info: version: '2.48.2', revision: '41bccdd10cf2c0560f637404c2d96164b67d9d67', time: '2015-10-09 13:08:06'
System info: host: '45.112.138-broadband.acttv.in', ip: '192.168.0.252', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.11.1', java.version: '1.8.0_65'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities [{applicationCacheEnabled=false, rotatable=false, chrome={userDataDir=/var/folders/zd/xg2w7l2d0gjdxh9ck8q3nlzr0000gn/T/.org.chromium.Chromium.mOuNkY}, takesHeapSnapshot=true, databaseEnabled=false, handlesAlerts=true, version=47.0.2526.80, platform=MAC, browserConnectionEnabled=false, nativeEvents=true, acceptSslCerts=true, locationContextEnabled=true, webStorageEnabled=true, browserName=chrome, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true}]
Session ID: 60d6e0f32aac1a1e865be086e955662c]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.WebDriverException: unknown error: Element is not clickable at point (524, 613). Other element would receive the click: <div class="vpopupWrapper nonner" id="v-popup" style="opacity: 0; display: block;">...</div>
  (Session info: chrome=47.0.2526.80)
  (Driver info: chromedriver=2.9.248307,platform=Mac OS X 10.11.1 x86_64) (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 44 milliseconds
Build info: version: '2.48.2', revision: '41bccdd10cf2c0560f637404c2d96164b67d9d67', time: '2015-10-09 13:08:06'
System info: host: '45.112.138-broadband.acttv.in', ip: '192.168.0.252', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.11.1', java.version: '1.8.0_65'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities [{applicationCacheEnabled=false, rotatable=false, chrome={userDataDir=/var/folders/zd/xg2w7l2d0gjdxh9ck8q3nlzr0000gn/T/.org.chromium.Chromium.mOuNkY}, takesHeapSnapshot=true, databaseEnabled=false, handlesAlerts=true, version=47.0.2526.80, platform=MAC, browserConnectionEnabled=false, nativeEvents=true, acceptSslCerts=true, locationContextEnabled=true, webStorageEnabled=true, browserName=chrome, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true}]
Session ID: 60d6e0f32aac1a1e865be086e955662c
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:422)
	at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:206)
	at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:158)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:647)
	at org.openqa.selenium.remote.RemoteWebElement.execute(RemoteWebElement.java:326)
	at org.openqa.selenium.remote.RemoteWebElement.click(RemoteWebElement.java:84)
	at com.vedantu.test.sanity1_testsuite.TakeSession.TakingSession(TakeSession.java:87)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.retryFailed(Invoker.java:1049)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1254)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:295)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:90)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:203)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:155)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.WebDriverException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TakingSession -->
      </class> <!-- com.vedantu.test.sanity1_testsuite.TakeSession -->
      <class name="com.vedantu.test.sanity1_testsuite.ListUsersSession">
        <test-method status="SKIP" signature="preCondition()[pri:0, instance:com.vedantu.test.sanity1_testsuite.ListUsersSession@6442b0a6]" name="preCondition" is-config="true" duration-ms="0" started-at="2015-12-11T15:21:09Z" finished-at="2015-12-11T15:21:09Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- preCondition -->
        <test-method status="SKIP" signature="ListUserSchedule()[pri:0, instance:com.vedantu.test.sanity1_testsuite.ListUsersSession@6442b0a6]" name="ListUserSchedule" duration-ms="0" started-at="2015-12-11T15:21:09Z" finished-at="2015-12-11T15:21:09Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ListUserSchedule -->
        <test-method status="SKIP" signature="postCondition()[pri:0, instance:com.vedantu.test.sanity1_testsuite.ListUsersSession@6442b0a6]" name="postCondition" is-config="true" duration-ms="0" started-at="2015-12-11T15:21:09Z" finished-at="2015-12-11T15:21:09Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- postCondition -->
      </class> <!-- com.vedantu.test.sanity1_testsuite.ListUsersSession -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
